python/datasources/acs_population.py

```diff
@@ -129,39 +129,29 @@ def update_col_types(frame):
     return frame
 
 
-class ACSPopulationBase(DataSource):
+class ACSPopulationIngester():
     """American Community Survey population data in the United States from the
        US Census."""
 
-    def __init__(self, county_level):
-        # TODO pass this in from message data.
-        self.base_acs_url = "https://api.census.gov/data/2019/acs/acs5"
+    def __init__(self, county_level, base_acs_url):
+        self.base_acs_url = base_acs_url
         """The base ACS url to use for API calls."""
```

Comment:
I don't think it's common practice to write docstring-style comments underneath member variables. Could you move the comments to above their variables and use '#' style comments instead?

Reply:
Huh I thought I read this somewhere. Oh well, updated to # style